src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,17 @@
+$bgcColor: #00acdc;
+$genCardBgc: #fff;
+$minorColor: #616070;
+$defaultColor: #060b35;
+$borderCardColor: #f3f3f3;
+
+$cardWidth: 200px;
+$gapX: 48px;
+$gapY: 46px;
+$columnNumb: 4;
+$mainWidth: $columnNumb * $cardWidth + ($columnNumb - 1) * $gapX;
+
+$imgBottom: 40px;
+$descBottom: 16px;
+$nameBottom: 4px;
+$codeBottom: 16px;
+$raitBottom: 24px;
```

Comment:
You should create variables only for repeated values, if you use them only once, don't create variables